### ISSUE TITLE
SSO with custom UI for multiple application

### ISSUE THREAD

**Issue by codeonnn** [IdentityServer/IdentityServer3#3357]
We are building SSO application with 2factor authentication using google app for MFA.

1. App1 UI (Team A)
2. App2 UI (Team A)
3. App3 Auth Login UI with registration and Profile management feature.  (Team A)
3. Authentication API (Team B)

Here Team B mostly works on API and responsible for providing authN/AuthX API and proposed.
a. We will create a IdentityServer3 with Implicit flow on which client app can be configured.
and will have to use the Login page of IdentityServer3, we will put a custom theme in Idser login page.
Team A to create App3 for profile update, management and registration.

b. But here login URL will be different and Profile management Url will be different.
which we don't want.

So Team B proposed no login page of IdSer with ResourceOwnerFlow,
Team A to hit Idser token endpoint from App3 with  user credentials input on App3.
But I think we cannot provide true single sign on for App1 and App2 using ResourceOwnerFlow, just not sure how to integrate these API with custom UI.

Can anybody point me to any other flow or samples by which we can have custom login page not of IdServ but still provide implicit flow by just hiting IdSer API's

Basically Team A mostly develops UI in NodeJs and Team B uses .NET for implementing API,
So cannot share a repo to build everything login and profile management in App3.

**Comment by leastprivilege** [comment]
This seems to be a general question about IdentityServer - not a bug report or an issue. 

Please use StackOverflow for that. This has the advantage that questions and answers can be easily found by search engines, and that there are more people answering questions than just us.

For IdentityServer3
https://stackoverflow.com/questions/tagged/?tagnames=identityserver3&sort=newest

For IdentityServer4
https://stackoverflow.com/questions/tagged/?tagnames=identityserver4&sort=newest

For commercial support options - visit
https://identityserver.io
